src/validation.cpp

```diff
@@ -1318,16 +1318,14 @@ int GetSpendHeight(const CCoinsViewCache& inputs)
     return pindexPrev->nHeight + 1;
 }
 
-namespace Consensus {
-bool CheckTxInputs(const CTransaction& tx, CValidationState& state, const CCoinsViewCache& inputs, int nSpendHeight)
+bool Consensus::CheckTxInputs(const CTransaction& tx, CValidationState& state, const CCoinsViewCache& inputs, int nSpendHeight, CAmount& nFees)
```

Comment:
Rather than forcing a bogus `nFees` (or unused) everywhere,  just wrap this function with a unused `nFees` stack variable.
Cleaner,  and less diff.